src/react/actions/__tests__/s3object.test.jsx

```diff
@@ -264,6 +286,134 @@ describe('s3object actions', () => {
+            fn: actions.putObjectMetadata(BUCKET_NAME, '', OBJECT_KEY, METADATA),
+            storeState: initState,
+            expectedActions: [
+                gettingObjectMetadataNetworkAction,
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.GET_OBJECT_METADATA_SUCCESS_ACTION(BUCKET_NAME, '', OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
+                dispatchAction.NETWORK_END_ACTION,
+                dispatchAction.NETWORK_END_ACTION,
+            ],
+        },
+        {
+            it: 'putObjectMetadata: should return expected actions -> test with prefix parameter',
+            fn: actions.putObjectMetadata(BUCKET_NAME, PREFIX, OBJECT_KEY, METADATA),
+            storeState: initState,
+            expectedActions: [
+                gettingObjectMetadataNetworkAction,
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.GET_OBJECT_METADATA_SUCCESS_ACTION(BUCKET_NAME, PREFIX, OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
+                dispatchAction.NETWORK_END_ACTION,
+                dispatchAction.NETWORK_END_ACTION,
+            ],
+        },
+        {
+            it: 'putObjectMetadata: should handle error -> test without prefix parameter',
+            fn: actions.putObjectMetadata(BUCKET_NAME, '', OBJECT_KEY, METADATA),
+            storeState: errorZenkoState(),
+            expectedActions: [
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.HANDLE_ERROR_SPEC_ACTION('The server is temporarily unavailable.'),
+                dispatchAction.NETWORK_END_ACTION,
+            ],
+        },
+        {
+            it: 'putObjectMetadata: should handle error -> test with prefix parameter',
+            fn: actions.putObjectMetadata(BUCKET_NAME, PREFIX, OBJECT_KEY, METADATA),
+            storeState: errorZenkoState(),
+            expectedActions: [
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.HANDLE_ERROR_SPEC_ACTION('The server is temporarily unavailable.'),
+                dispatchAction.NETWORK_END_ACTION,
+            ],
+        },
+        {
+            it: 'putObjectTagging: should return expected actions -> test without prefix parameter',
+            fn: actions.putObjectTagging(BUCKET_NAME, '', OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
+            storeState: initState,
+            expectedActions: [
+                gettingObjectTagsNetworkAction,
+                gettingObjectMetadataNetworkAction,
+                dispatchAction.GET_OBJECT_METADATA_SUCCESS_ACTION(BUCKET_NAME, '', OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
+                dispatchAction.NETWORK_END_ACTION,
+                dispatchAction.NETWORK_END_ACTION,
+            ],
+        },
+        {
+            it: 'putObjectTagging: should return expected actions -> test with prefix parameter',
+            fn: actions.putObjectTagging(BUCKET_NAME, PREFIX, OBJECT_KEY, HEAD_OBJECT_RESPONSE, []),
```

Comment:
Can we provide a non-empty tags array? (type `Tags`)

Reply:
done